### DISCUSSION TITLE
same deployment, different schedules, different parameters

### DISCUSSION THREAD

**Discussion by ayushxx7** [PrefectHQ/prefect#16156] [Q&A]
Is there a way to setup a prefect flow / deployment in such a way that it can be run on multiple schedules with different parameters?
I have a `update` job that should
- update 100 top queries daily
- update 500 top queries every 3 days
- update all queries every week

I see there is 'CronSchedule' but I don't know how to set parameters linked with each schedule. 

I think this discussion is same as https://github.com/PrefectHQ/prefect/discussions/10161 but that one is unanswered still.

**Comment by aaazzam** [comment]
Currently no, but would like to make this happen. 

Want to upvote this one? https://github.com/PrefectHQ/prefect/issues/14524
